tap-setup-scripts/src/inputs/tap-values.yaml

```diff
@@ -5,22 +5,34 @@ ceip_policy_disclosed: true # Installation fails if this is set to 'false'
 buildservice:
   kp_default_repository: #@ data.values.repositories.build_service
   kp_default_repository_aws_iam_role_arn: #@ data.values.buildservice.buildservice_arn
-  tanzunet_username: #@ data.values.tanzunet.username
```

Comment:
unused variables, hence removed

Reply:
Thanks for the additional context, @satya-dillikar. 😃